cypress/e2e/api-delegations.cy.tsx

```diff
@@ -1,6 +1,8 @@
-describe('renders api-delegations', () => {
+import { RouterPath } from '@/routes/Router';
+
+describe(RouterPath.GivenApiDelegations + '/' + RouterPath.ReceivedApiDelegations, () => {
   it('renders correctly', () => {
-    cy.visit('/api-delegations');
+    cy.visit(RouterPath.GivenApiDelegations + '/' + RouterPath.ReceivedApiDelegations);
```

Comment:
I might be misunderstanding something here, but why is the route used here `given-api-delegations/received-api-delegations` ?

Reply:
No you're not. I wobbled with the the urls here. Nice spotted.